userguide/virtualmachines.rst

```diff
@@ -44,85 +44,112 @@ output shows the POPCNT feature, this processor can be used with
    processors ceased in 2012 or 2013.
 
 
-.. index:: Creating VMs
-.. _Creating VMs:
+.. index:: Virtual Machines Page
+.. _Virtual Machines Page:
 
-Creating VMs
-------------
+Virtual Machines Page
+---------------------
 
-Click on
+Go to
 :menuselection:`Virtual Machines`
-and |ui-add| to open the wizard shown in
-:numref:`Figure %s <vms_add_fig>`:
+to see a table with each installed VM. A graph of total VM memory usage
+displays at the top of the page.
+
+.. figure:: images/virtual-machines.png
 
+   Virtual Machines Page
 
-.. _vms_add_fig:
 
-.. figure:: images/virtual-machines-add-wizard-type.png
+The VM :guilabel:`Name`, :guilabel:`State`, :guilabel:`VNC Port` (if
+present), :guilabel:`Type`, :guilabel:`Description`, number of
+:guilabel:`Virtual CPUs`, :guilabel:`Memory Size` (in MiB),
+:guilabel:`Boot Loader Type`, and :guilabel:`Autostart` values are shown
+in the table. The table display is adjusted by clicking
+:guilabel:`COLUMNS` and selecting which columns are shown in the table.
 
-   Add VM
+Click |ui-options| to see additional buttons for a VM.
 
+These choices are available in the |ui-options| menu regardless of the
+current :guilabel:`State`:
 
-Create a virtual machine by following these steps in the creation
-wizard:
+* :guilabel:`Edit` opens a form to change the VM settings. The VM name,
+  description, number of virtual CPUs, allocated memory, boot loader,
+  and option to start the VM on system boot can all be adjusted.
 
-#. **Select VM wizard type**
+* :guilabel:`Delete` removes the VM. :ref:`Zvols <Adding Zvols>` used in
+  :ref:`disk devices <vms-disk-device>` and image files used in
+  :ref:`raw file <vms-raw-file>` devices are *not* removed when a VM
+  is deleted. These resources can be removed manually in
+  :menuselection:`Storage --> Pools` after it is determined that the
+  data in them has been backed up or is no longer needed.
 
-   Select *Virtual Machine (VM)* or
-   *Docker Host*.
+* :guilabel:`Devices` is used to add, remove, and edit the VM devices.
 
-#. **Operating System**
+* :guilabel:`Clone` makes a copy of the VM. The new clone has
+  :samp:`_clone{N}` added to the name, where :samp:`{N}` is the number
+  of the clone. %brand% displays a substantive error message if cloning
```

Comment:
```suggestion
  number.
```
"Displays an error message if an error occurs" is not really useful, it should be a safe assumption by the reader that this happens everywhere.